### ISSUE TITLE
Is Iridium dead? 

### ISSUE THREAD

**Issue by Anutrix** [ParticleCore/Iridium#882]
There hasn't been a commit to master branch since April 2020 which was more than 2 years ago. So, does this mean that this project is dead?! 

If yes, is there any open-source alternative or equivalent of this?

**Comment by NekoAlosama** [comment]
Seems so, or at least @ParticleCore is busy IRL. I don't current;y know of any good alternatives, but you should be able to find one after some searching
![firefox_Fc0NDbYpL8](https://user-images.githubusercontent.com/47763383/188443809-3ff77cbc-ff02-4cff-837e-807693bf8637.png)

**Comment by raingart** [comment]
>I don't currenty know of any good alternatives

https://openuserjs.org/scripts/raingart/Nova_YouTube

**Comment by ParticleCore** [comment]
Please don't use this issue tracker to publicize others' products. The project is alive, I just haven't made it public, only been using it privately so far.

**Comment by ImprovedTube** [comment]
hi @ParticleCore, i dont know if you are committed to saying this

**Comment by ParticleCore** [comment]
> hi @ParticleCore, i dont know if you are committed to saying this

I am afraid I don't understand what you said.

**Comment by ImprovedTube** [comment]
i guess you might wish to reveal the new version already and/or undelete the "competitor"

**Comment by ImprovedTube** [comment]
i guess you might wish to reveal the new version already and/or undelete "competitor"-comments

**Comment by ImprovedTube** [comment]
how are you?

**Comment by ParticleCore** [comment]
The reason for limiting references to other products is to avoid falling in the situation of having users requesting support or reporting problems on those products in a issue tracker that is not related to them. This is a situation that happened in the past multiple times and mostly happened by something as simple as someone proposing an alternative client, others using it as a result, and later on coming back to that issue complaining about problems with that product which is not Iridium.

As for the new version it is only being used privately for now.

Please keep the issues in this tracker related to Iridium.

Hope this was clarifying enough, closing for now since it has gone off track.

**Comment by ImprovedTube** [comment]
@ParticleCore 😁 yes!  You dont need to serve users one by one. 
Nor let them confuse/mix stuff here.  

 And you _could_ even keep work private for yourself.  Often times that would be protesting how it was before or your experience with it. 
(Whether you mean it or not or subconsciously
 - thats just the effect it might have on those who might still notice.)
And all of it is optional.  
A lot of 'conventions' / 'best practices'  are actually just personal.
(i.e. how to 'run' a project)

> off track

Not by my definition. A [bit] wider context.
1. @Anutrix spoke for a (large) group of people & your fans.

2. For users & usability it makes sense to mix different projects.
 
Instead of saying developers run projects,
we can also say people fill marketgaps.
The gaps are there without us.
 We are solving them and sometimes complifying them too.
Often just really starting and stopping for personal situations,
 instead of the world and our talent or passion.

Today & together the `"top 100 open Browser Extensions for YouTube (+userscripts)
`(without adblockers, returnDislikes & sponsorblock)"
Might "only" serve below 3 million users?
- And they cost maybe only 100 hours of developer work per day.
Even it is a pretty redundant (inefficient), communicationless way.
Each feature just needs to exist once, in one repo, 
while the 10 different projects wrapped around it, then could be summed up as "various themes, documentation, legacy GUI"
So the quality and usability and UX would all skyrocket.

**in other words**, 
 why doesnt wikipedia support two articles about the same thing,
But github supports two repos about the same thing? And what about NPMs for example, how many in 3.2 million  are unique?

  - While maybe the more active 40% of our the users wouldn't mind at all to pay $1 per year, in average for a license including support for all these projects. if that would be possible, so 
     - While the developers could vote to donate much of this money for human rights but also be able to hire themselves at need or hire assitants, backup developers, repo managers.

so what are / were we doing?  
Would you call this optimistic or..?

**Comment by ImprovedTube** [comment]
@ParticleCore 😁 yes!  You dont need to serve users one by one. 
Nor let them confuse/mix stuff here.  
  
 And you _could_ even keep work private for yourself.  Often times that would be protesting how it was before or your experience with it. 
(Whether you mean it or not or subconsciously
 - thats just the effect it might have on those who might still notice.)
And all of it is optional.  
A lot of 'conventions' / 'best practices'  are actually just personal.
(i.e. how to 'run' a project)

> off track

Not under a  [bit] wider definition context. 

1. @Anutrix spoke for a (large) group of people & your fans.

2. For users & usability it makes sense to mix different projects.
 
Instead of saying developers run projects,
we can also say people fill marketgaps.
The gaps are there without us.
 We are solving them and sometimes complifying them too.
Often just really starting and stopping for personal situations,
 instead of the world and our talent or passion.

Today & together the `"top 100 open Browser Extensions for YouTube (+userscripts)
`(without adblockers, returnDislikes & sponsorblock)"
Might "only" serve below 3 million users?
- And they cost maybe only 100 hours of developer work per day.
Even it is a pretty redundant (inefficient), communicationless way.
Each feature just needs to exist once, in one repo, 
while the 10 different projects wrapped around it, then could be summed up as "various themes, documentation, legacy GUI"
So the quality and usability and UX would all skyrocket. <br> **in other words**, 
 why doesnt wikipedia support two articles about the same thing,
But github supports two repos about the same thing? And what about NPMs for example, how many in 3.2 million  are unique?

  - While maybe the more active 40% of our the users wouldn't mind at all to pay $1 per year, in average for a license including support for all these projects. if that would be possible, so 
     - While the developers could vote to donate much of this money for human rights but also be able to hire themselves at need or hire assitants, backup developers, repo managers.

so what are / were we doing?  
Would you call this optimistic or..?

**Comment by ParticleCore** [comment]
Sorry, forgot to lock the issue, there is nothing useful being added to the issue or the extension itself.
